hack/util.sh

```diff
@@ -67,15 +67,14 @@ function util::get_target_source() {
 #  - $1: package name, such as "sigs.k8s.io/controller-tools/cmd/controller-gen"
 #  - $2: package version, such as "v0.4.1"
 function util::install_tools() {
-	local package="$1"
```

Comment:
irrelevant changes

Reply:
if you think this format change is necessary, plz commit it in another pr

Reply:
I have reversed this.